src/templates/components/facilityListing/index.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { FacilityListing } from './index'
+
+describe('FacilityListing with valid data', () => {
+  test('renders FacilityListing component', () => {
+    expect(true).toBe(true)
+  })
+})
```

Comment:
Prolly add some basic smoke tests in here to make sure each part of the component does what's expected. (E.g. renders the title, operating status flags, address, etc. when they should be rendered, don't render them when they shouldn't be rendered. That kind of thing.)

Reply:
Forgot that I hadn't done that yet. Pushed an update